source/js/foundation/template-js-handler/article-hero-video-handler.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Adding play/pause functionality to the optional Article Page hero video section.
+ */
+
+ const articleHeroVideoHandler = () => {
```

Comment:
If we don't need to preserve the declare-time `this`, we don't need arrow functions.

```suggestion
function articleHeroVideoHandler() {
```